examples/curio/curio-server.py

```diff
@@ -190,6 +191,13 @@ def __init__(self, sock, root):
 
 
 if __name__ == '__main__':
-    kernel = Kernel()
-    kernel.add_task(h2_server(('', 8080), sys.argv[1]))
+    host = sys.argv[2] if len(sys.argv) > 2 else "localhost"
+    kernel = Kernel(with_monitor=True)
+    kernel.add_task(h2_server((host, 5000),
+                              sys.argv[1],
+                              "{}.crt.pem".format(host),
+                              "{}.key".format(host)))
+    print("Try GETting:")
+    print("    OSX after 'brew install curl --with-c-ares --with-libidn --with-nghttp2 --with-openssl':")
+    print("/usr/local/Cellar/curl/7.47.1/bin/curl --tlsv1.2 --http2 -k https://localhost:5000/bundle.js")
```

Comment:
Ok, but it can be hard to tell what is actually happening with browser. In fact, Safari also hangs with this, and it's hard to tell on what file it got hung.